code/game/objects/structures/extinguisher.dm

```diff
@@ -14,9 +14,6 @@
 /obj/structure/extinguisher_cabinet/Initialize(mapload, ndir, building)
 	. = ..()
 	if(building)
-		setDir(ndir)
-		pixel_x = (dir & 3)? 0 : (dir == 4 ? -27 : 27)
-		pixel_y = (dir & 3)? (dir ==1 ? -30 : 30) : 0
```

Comment:
what does removing this do even

Reply:
It pixel shifts the wall structure to actually be on the wall when placed. It's unnecessary however, as the wall frame item already has a variable that can handle that.